### ISSUE TITLE
Orange label near nickname

### ISSUE THREAD

**Issue by sybot99** [ChatSecure/ChatSecure-iOS#918]
I don't know what it is the issue that icon change green -> orange near nickname in app. But I find a bug when messange dont come to my friend. The option in contact is strongly OTR

I want to try the OMEMO, but I dont how to change OTR to OMEMO, because the option is grey and I can it turn on

**Comment by chrisballinger** [comment]
The person on the other side must also support OMEMO or it won't work. Can they send you OMEMO messages?

The green/orange is unrelated and is the user's "availability", where green means active and orange means away.

**Comment by sybot99** [comment]
We both use ChatSecure on iOS

**Comment by sybot99** [comment]
Why messages dont delivered? 

https://s8.hostingkartinok.com/uploads/images/2018/01/d6d71b2683930b03982890e2fff22194.png

**Comment by chrisballinger** [comment]
Messages won't be delivered if the other user has the app in the background. Once the app is removed from memory the OTR session is destroyed. This is why OTR sucks on mobile and you must use OMEMO.

If you make a new account does OMEMO work?

**Comment by sybot99** [comment]
Now I understand about OTR. It's only for computer use.

>If you make a new account does OMEMO work?
I just register two new accounts on exploit.im and add them in two iPhones 8 plus. OMEMO still grey 
http://prntscr.com/hw5qkx
http://prntscr.com/hw5rdl

**Comment by chrisballinger** [comment]
Try another server from near the top of this list: https://conversations.im/compliance/

**Comment by sybot99** [comment]
Yes, it works perfect. Don't you know server without logs and with OMEMO ?
I test jabber.cat but it has logs
